app/views/homepages/search_results.html.erb

```diff
@@ -0,0 +1,33 @@
+<!-- List View:
+
+This shall show a paged list of recipes for a given search term, ten at a time
+This view shall show the name of the recipe and the corresponding photo
+The view shall have a link from the recipe to a recipe show view -->
+
+<h1 class="center-this">Search Results</h1>
+
+<% if @results == [] %>
+  <section class="center-this no-results">
+    <p>No results found</p>
+    <p>😐</p>
+  </section>
+<% else %>
+  <section class="results">
+      <% @results.each do |recipe| %>
+      <div class="row">
+        <div class="small-12 medium-6 large-5 column center-this">
+          <%= image_tag recipe.image %>
+        </div>
+        <div class="small-12 medium-6 large-7 column text">
+            <p><%= link_to recipe.name, show_path(recipe_uri: recipe.uri) %></p>
```

Comment:
I'd suggest making the image a link as well.  It's natural to click on the image to get to the show page.